src/modules/home/api/create-accommodation.ts

```diff
@@ -0,0 +1,41 @@
+import { useMutation } from '@tanstack/react-query';
+import type { AccommodationFormData } from '#/modules/home/types/accommodation-form-data.interface.ts';
+import { ENDPOINTS } from '#/modules/home/constants/endpoints.constant.ts';
+import { toast } from 'react-toastify';
+import { type Accommodation } from '#/modules/home/types/accommodation.type.ts';
+import { api } from '#/configs';
+import { type AxiosError } from 'axios';
+
+export const useCreateAccommodation = () => {
+  api.defaults.headers['Content-Type'] = 'multipart/form-data';
```

Comment:
i recommend not to change our axios client defaults because it is used in other requests also so something might break. you can pass the header when you make the request, right?